CONTRIBUTING.md

```diff
@@ -37,21 +62,21 @@ printed.
 ### Testing and Continuous Integration (CI)
 
 > [!IMPORTANT]
-> Don't run tests with `cargo test`. Rather use [./run_tests.sh](./run_tests.sh).
-> Or run the command contained therein:
->
-> `cargo test --lib --tests -- --skip sourcegen_ast --skip sourcegen_ast_nodes`
+All pull requests must pass CI. To run the full suite locally, use:
```

Comment:
You should also mention that CI runs `just check_sourcegen` separately from the `just ci` recipe.